api_examples/graphql/clients/python/run_controls/requirements.txt

```diff
@@ -5,5 +5,5 @@ idna==2.10
 requests==2.24.0
 urllib3==1.25.10
 xdg==3.0.2
-PyYAML==5.3
-../turbot
\ No newline at end of file
+PyYAML==5.3.1
+# ../turbot
```

Comment:
Why did you remove this? This is what points all the python scripts to pick up the single turbot.py

The reason we have it centrally and not on each branch is due to the fact that if you change it in one, then you will have to change it in all other folders and this is hard to keep track of. 

Contact me if there are valid reasons for changing this and then we can address the issue.